src/apps/defiedge/common/defiedge.strategy.token-fetcher.ts

```diff
@@ -109,24 +131,62 @@ export abstract class DefiedgeStrategyTokenFetcher extends AppTokenTemplatePosit
     return sharePrice;
   }
 
+  async getBalances(_address: string): Promise<AppTokenPositionBalance<DefiedgeStrategyTokenDataProps>[]> {
```

Comment:
You shouldn't have to override `getBalances`
What's wrong with the current implementation ?

Reply:
My Bad I though it was required. I have reverted the commit